content/api/relay-webhooks.apib

```diff
@@ -325,6 +278,16 @@ Create a relay webhook by providing a **relay webhooks object** as the POST requ
 
 ### Create an SMPP Relay Webhook [POST]
```

Comment:
It's weird that this doc lets examples guide the structure (like transmissions). This is the just the same call as SMTP but with different options. I think this is well intentioned but belongs in explanatory docs.